site2/website-next/docs/tiered-storage-filesystem.md

```diff
@@ -12,307 +12,526 @@ This chapter guides you through every step of installing and configuring the fil
 
-   * If you are running Pulsar in a bare metal cluster, make sure that `offloaders` tarball is unzipped in every broker's Pulsar directory.
-   * If you are running Pulsar in Docker or deploying Pulsar using a Docker image (such as K8S and DCOS), you can use the `apachepulsar/pulsar-all` image instead of the `apachepulsar/pulsar` image. `apachepulsar/pulsar-all` image has already bundled tiered storage offloaders.
+    :::note
+    
+    * If you run Pulsar in a bare metal cluster, ensure that the `offloaders` tarball is unzipped in every broker's Pulsar directory.
+    * If you run Pulsar in Docker or deploying Pulsar using a Docker image (such as K8S and DCOS), you can use the `apachepulsar/pulsar-all` image. The `apachepulsar/pulsar-all` image has already bundled tiered storage offloaders.
 
-   :::
+    :::
 
 3. Copy the Pulsar offloaders as `offloaders` in the Pulsar directory.
 
-   ```
-   
-   mv apache-pulsar-offloaders-2.5.1/offloaders apache-pulsar-2.5.1/offloaders
-
-   ls offloaders
-   
-   ```
-
-   **Output**
+    ```
+    mv apache-pulsar-offloaders-2.5.1/offloaders apache-pulsar-2.5.1/offloaders
 
-   ```
-   
-   tiered-storage-file-system-2.5.1.nar
-   tiered-storage-jcloud-2.5.1.nar
-   
-   ```
+    ls offloaders
+    ```
 
-   :::note
+    **Output**
 
-   * If you are running Pulsar in a bare metal cluster, make sure that `offloaders` tarball is unzipped in every broker's Pulsar directory.
-   * If you are running Pulsar in Docker or deploying Pulsar using a Docker image (such as K8s and DCOS), you can use the `apachepulsar/pulsar-all` image instead of the `apachepulsar/pulsar` image. `apachepulsar/pulsar-all` image has already bundled tiered storage offloaders.
+    ```
+    tiered-storage-file-system-2.5.1.nar
+    tiered-storage-jcloud-2.5.1.nar
+    ```
 
-   :::
+    :::note
+    
+    * If you run Pulsar in a bare metal cluster, ensure that `offloaders` tarball is unzipped in every broker's Pulsar directory.
+    
+    * If you run Pulsar in Docker or deploying Pulsar using a Docker image (such as K8s and DCOS), you can use the `apachepulsar/pulsar-all` image. The `apachepulsar/pulsar-all` image has already bundled tiered storage offloaders.
 
+    :::
 ## Configuration
 
 :::note
 
-Before offloading data from BookKeeper to filesystem, you need to configure some properties of the filesystem offloader driver. 
+Before offloading data from BookKeeper to filesystem, you need to configure some properties of the filesystem offloader driver.
 
 :::
 
 Besides, you can also configure the filesystem offloader to run it automatically or trigger it manually.
 
 ### Configure filesystem offloader driver
 
-You can configure filesystem offloader driver in the configuration file `broker.conf` or `standalone.conf`.
+You can configure the filesystem offloader driver in the `broker.conf` or `standalone.conf` configuration file.
 
 - **Required** configurations are as below.
+
+    <!--DOCUSAURUS_CODE_TABS-->
+
+    <!--HDFS-->
```

Comment:
When adding multiple tabs for docs in `website-next`, please adhere to the new syntax: https://docs.google.com/document/d/1IV35SI_F8G8cL-Vuzknc6RTGLK9_edRMpZpnrHvAWNs/edit#bookmark=id.xmxlbpg3yug1

You can take this (https://github.com/apache/pulsar/blob/master/site2/website-next/versioned_docs/version-2.7.1/admin-api-namespaces.md) as a reference